src/Suave/Owin.fs

```diff
@@ -686,7 +687,9 @@ module OwinApp =
 
       let impl (conn, response) : SocketOp<unit> = socket {
         // disposable because it buffers what the OwinApp writes to the stream
-        use wrapper = new OwinDictionary({ ctx with response = response })
+        // also, OWIN expects the default HTTP status code to be 200 OK
+        // could also set the status code after calling SocketOp.ofAsync if the value is not set
+        use wrapper = new OwinDictionary({ ctx with response = { response with status = HTTP_200 } })
```

Comment:
If by "succeed" you mean "complete," then yes. However, the default for OWIN is `200 OK`. So if you mount an OWIN app that does nothing but complete, the response should send back `200 OK`.